app/views/home/_user_navbar.html.erb

```diff
@@ -1,6 +1,6 @@
 <div class="user_navbar">
   <div class="user_navbar__box">
-    <%= image_tag('profile.png', class: "user_navbar__image") %>
+   <%#  <%= image_tag('profile.png', class: "user_navbar__image") %>
```

Comment:
If you don't occupy this line, delete it. Do not leave commented code on GitHub.

Reply:
Done